readthedocs/projects/models.py

```diff
@@ -352,9 +352,20 @@ class Project(models.Model):
         default='',
         blank=True,
         help_text=_(
-            'Path from project root to <code>conf.py</code> file '
-            '(ex. <code>docs/conf.py</code>). '
-            'Leave blank if you want us to find it for you.',
+            "Path from project root to <code>conf.py</code> file "
+            "(ex. <code>docs/conf.py</code>). "
+            "Leave blank if you want us to find it for you.",
+        ),
+    )
+    rtd_conf_file = models.CharField(
+        _(".readthedocs.yml configuration file"),
+        max_length=255,
+        default="",
+        blank=True,
+        help_text=_(
+            "Path from project root to <code>.readthedocs.yml</code> file "
+            "(ex. <code>docs/.readthedocs.yml</code>). "
+            "Leave blank if you want us to find it for you.",
```

Comment:
I think we probably don't want to have any finding logic, and just support top-level and explicitly defined  values.

Reply:
This is copy-paste from `conf_py_file`, I don't think it's intentional :) Will update :+1: